lib/evaluate/host.h

```diff
@@ -39,10 +39,14 @@ class HostFloatingPointEnvironment {
 public:
   void SetUpHostFloatingPointEnvironment(FoldingContext &);
   void CheckAndRestoreFloatingPointEnvironment(FoldingContext &);
+  bool HasSubnormalFlushingHardwareControl() {
+    return HasSubnormalFlushingHardwareControl_;
+  }
 
 private:
   std::fenv_t originalFenv_;
   std::fenv_t currentFenv_;
+  bool HasSubnormalFlushingHardwareControl_{false};
```

Comment:
If you want your code to look more like the project style, it would be better to not capitalize the data member or its accessor.

Reply:
Thanks, done